test/versioned/undici/requests.tap.js

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright 2021 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const tap = require('tap')
+const { DESTINATIONS } = require('../../../lib/config/attribute-filter')
+const helper = require('../../lib/agent_helper')
+const metrics = require('../../lib/metrics_helper')
+
+tap.test('Undici request tests', (t) => {
+  t.autoend()
+
+  let agent
+  let undici
+
+  t.before(() => {
+    agent = helper.instrumentMockedAgent({
+      feature_flag: {
+        undici_instrumentation: true
+      }
+    })
+
+    undici = require('undici')
+  })
+
+  t.teardown(() => {
+    helper.unloadAgent(agent)
+  })
+
+  t.test('should not fail if request not in a transaction', async (t) => {
+    const { statusCode } = await undici.request('https://httpbin.org', {
```

Comment:
We've typically been using example.com. Looks like httpbin serves a similar purpose so I'm fine with it as long as you have some confidence it is going to be as reliable as example.com as far as not getting denied for too many requests, going down, etc.

Reply:
it should be fine, i've used it historically for CI stuff and it was very stable.  i only used it because it allows you to affect diff responses.